cmd/query.go

```diff
@@ -114,6 +107,13 @@ func runQueryCmd(cmd *cobra.Command, args []string) {
 	// convert the query or sql file arg into an array of executable queries - check names queries in the current workspace
 	queries := execute.GetQueries(args, workspace)
 
+	clientReadyChannel := make(chan bool)
+
+	// go func() {
```

Comment:
why not in prerun?

Reply:
this is not required anymore. removed and fixed